openmrs/api/src/main/java/org/projectbuendia/openmrs/api/db/hibernate/HibernateProjectBuendiaDAO.java

```diff
@@ -31,4 +37,21 @@ public SessionFactory getSessionFactory() {
     public void setSessionFactory(SessionFactory sessionFactory) {
         this.sessionFactory = sessionFactory;
     }
+
+    @Override
+    public List<Encounter> getEncountersCreatedAtOrAfter(Date date) {
+        // NOTES:
+        // - this code relies on the assumption that observations can't be modified independently of
+        // encounters.
+        // - This doesn't actually return encounters "modified on or after", it's currently
+        // encounters "created on or after". This is ok for our purposes because we don't have
+        // the ability to modify Encounters in Buendia.
```

Comment:
Actually, this assumption breaks with @G1DR4's change to void observations - once an observation can be voided, an encounter can logically be modified (because encounters have multiple observations). I'll need to adjust this logic when that change goes through.